### PR TITLE
Verify claim fee

### PR DESCRIPTION
**Pull request by vianvi**
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by vianvi** [comment]
as all the tests passed - merging pull request

File: src/tests/citizen/defence/pages/defendant-view-claim.ts [hunk 1]
```diff
@@ -13,6 +13,10 @@ export class DefendantViewClaimPage {
   }
 
   clickRespondToClaim (): void {
+    I.waitForText('View amount breakdown')
```
Comment (timja-kainos) [inline src/tests/citizen/defence/pages/defendant-view-claim.ts]: why do you need to wait for this text?

File: src/tests/citizen/defence/pages/defendant-view-claim.ts [hunk 2]
```diff
@@ -13,6 +13,10 @@ export class DefendantViewClaimPage {
...
+    I.click('summary')
```
Comment (timja-kainos) [inline src/tests/citizen/defence/pages/defendant-view-claim.ts]: clicking 'View amount breakdown' would be clearer could you do that instead please?

Comment (vianvi) [reply src/tests/citizen/defence/pages/defendant-view-claim.ts]: tried earlier- as the element is not a link text and codecept doesn't recognises it as link text

File: src/tests/citizen/defence/pages/defendant-view-claim.ts (hunk 1 above)
Comment (vianvi) [reply src/tests/citizen/defence/pages/defendant-view-claim.ts]: it will help for the test to load text inside summary(expandable area)

**Pull request merged**